package.json

```diff
@@ -40,14 +40,15 @@
         "ts-jest": "^23.1.4",
         "ts-node": "^7.0.1",
         "tslint": "^5.11.0",
-        "typescript": "^3.0.3"
+        "typescript": "^3.2.2"
```

Comment:
no to jest gruba zmiana. wydaje mi się, że angular 7 miał jakieś problemy z 3.2.2, więc zostawiłbym narazie 3.0.3 (którą domyślnie instaluje angula cli)

Reply:
robilem reinstall bo nie mogłem odpalić i po tym poszło, ale fakt nie zwróciłem uwagi na wersje tylko dałem najnowszą, jutro przywróce

Reply:
ERROR in The Angular Compiler requires TypeScript >=3.1.1 and <3.2.0 but 3.0.3 was found instead.